src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Methodologies/MethodologyControllerTests.cs

```diff
@@ -2,7 +2,6 @@
 using System.Collections.Generic;
 using GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.Methodologies;
 using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.Methodologies;
-using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
```

Comment:
Just curious as to how you went about this because if I use Rider and 'Optimise Imports' on, say, the Admin project, it fixes more missing imports (and adjusts others to remove more unnecessary code) than what you have on this PR so it might be worth doing that.

Reply:
I was going through the errors in Errors In Solution in Rider. Didn't know about "Optimise Imports" - I won't apply that to this PR, but next warnings-cleanup PR I make I'll do that.

Reply:
Ok cool